src/test/java/com/coherentsolutions/aqa/java/web/makarevich/YandexMailLoginPageObjectTest.java

```diff
@@ -0,0 +1,42 @@
+package com.coherentsolutions.aqa.java.web.makarevich;
+
+import com.coherentsolutions.aqa.web.makarevich.pages.Yandex360MailPage;
+import com.coherentsolutions.aqa.web.makarevich.pages.YandexMailLoginPage;
+import com.coherentsolutions.aqa.web.makarevich.pages.components.Header;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.coherentsolutions.aqa.web.makarevich.constants.TestConstants.YANDEX_MAIL_PASSWORD_2;
+import static com.coherentsolutions.aqa.web.makarevich.constants.TestConstants.YANDEX_MAIL_USERNAME_2;
+import static com.coherentsolutions.aqa.web.makarevich.constants.UrlConstants.YANDEX_MAIL_URL;
+
+public class YandexMailLoginPageObjectTest extends TestBase {
```

Comment:
Both of your tests classes uses same page objects. We can move it to TestBase in this case

Reply:
Done! Moved all page classes declaration and initialization to TestBase calss